reference/storage-class-parameters.md

```diff
@@ -49,7 +49,7 @@ The `agents.core.capacity.thin` spec present in the Mayastor helm chart consists
 {% hint style="info" %}
 Note:
 1. By default, the volumes are provisioned as `thick`. 
-2. For a pool of a particular size, say 10 Gigabytes, a volume > 10 Gigabytes cannot be created, as Mayastor 2.3.0 does not support pool expansion.
+2. For a pool of a particular size, say 10 Gigabytes, a volume > 10 Gigabytes cannot be created, as Mayastor 2.4.0 does not support pool expansion.
```

Comment:
We can remove version from here.
Just say, "as Mayastor currently does not support pool expansion"